gradle-sls-packaging-api/build.gradle

```diff
@@ -0,0 +1,10 @@
+apply plugin: 'com.palantir.external-publish-jar'
+apply plugin: 'org.inferred.processors'
+
+dependencies {
+    api 'com.fasterxml.jackson.core:jackson-databind'
+    api 'com.palantir.safe-logging:preconditions'
```

Comment:
I think both of those can be `implementation` given they are internal dependencies?